### ISSUE TITLE
capture_force & dialog_acquire conflicts

### ISSUE THREAD

**Issue by RogerLaw** [shin1m/alexaagent#3]
when capture_force is true, i think it means capture should start immediately and current dialog  should stop as well. however now it will wait for active dialog ends.

**Comment by shin1m** [comment]
Yes, currently, it interrupts Alerts and Content channels but Dialog channel.
This is because I was not sure that stopping a dialog is allowed in the [API](https://developer.amazon.com/public/solutions/alexa/alexa-voice-service/content/avs-api-overview).
But now I have found "Note: If a user barges in and interrupts Alexa, SpeechSynthesizer state should be cleared." in [SpeechSynthesizer Interface](https://developer.amazon.com/public/solutions/alexa/alexa-voice-service/reference/speechsynthesizer).
So, I will improve the behavior.

Thank you.

**Comment by shin1m** [comment]
Thank you for waiting.
I pushed the change to stop SpeechSynthesizer immediately when capture_force become true.
Please check it out.

**Comment by RogerLaw** [comment]
thank you! I just tried. the dialog was interrupted when button pressed, so this works.
When I tried 'what's my flash briefing', while it's reading the news, I pressed button, it won't stop immediately, and still playing for 2~4 seconds, then stop and take recognition.
And when I was playing iHeartradio station, when I pressed the button, console printed 'PAUSE' and when released, it printed 'RESUME', however music doesn't stop on 'PAUSE' and need to wait for a few seconds. beside, upon recognition finished, the music content and SpeechSynthesizer were playing in the same time.

**Comment by shin1m** [comment]
Oh I guess there must be buffered data remaining in OpenAL device.
I'll try to fix it.
Thank you.

**Comment by shin1m** [comment]
I pushed the change which pauses OpenAL source of AudioPlayer when interrupted.
Please check it out.

**Comment by RogerLaw** [comment]
sorry I just got time to test it. now it woks!
still one minor issue: when I interrupt the content and say "next" / or "play"+something else, it will still play 1 second of audio which was paused, then new replaced with new URL.

**Comment by shin1m** [comment]
Thank you for the report.
Currently, the content is resumed when the interruption ends, and continues playing until a new directive comes from AVS.
Let me consider whether the content should be cleared or not in this case.
Best regards,
